com.unity.shadergraph/Editor/Drawing/Blackboard/BlackboardProvider.cs

```diff
@@ -229,7 +229,10 @@ void AddProperty(IShaderProperty property, bool create = false, int index = -1)
                 return;
 
             if (create)
+            {
                 property.displayName = m_Graph.SanitizePropertyName(property.displayName);
+                property.overrideReferenceName = m_Graph.SanitizePropertyReferenceName(property.overrideReferenceName);
```

Comment:
This is kind of weird, as the `overrideReferenceName` will always be `null` here. Basically `create` means "the user created this just now and we should save it and focus the text field"